content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -103,38 +110,38 @@ Upgrading the Istio CNI agent to a compatible version in-place will not disrupt
 $ helm upgrade istio-cni istio/cni -n istio-system
 {{< /text >}}
 
-### Upgrade the Gateway component (optional)
+## Upgrade waypoints and gateways using tags
 
-Gateway components manage east-west and north-south dataplane traffic between ambient mesh boundaries, as well as some aspects of the L7 dataplane.
+If you have followed best practices, all of your gateway, workload, and namespaces use either the default revision (which is effectively a tag named default), or the `istio.io/rev` label with the value set to a tag name. We will now upgrade all of these data planes to the new version of Istio by moving their tags to point to the new version, one at a time. To list all tags in your cluster, run:
```

Comment:
```suggestion
If you have followed best practices, all of your gateways, workloads, and namespaces use either the default revision (effectively, a tag named `default`), or the `istio.io/rev` label with the value set to a tag name. You can now upgrade all of these to the new version of the Istio data plane by moving their tags to point to the new version, one at a time. To list all tags in your cluster, run:
```

Reply:
This is incorrect.  This step is only for gateways that are not managed by istiod.  Others are upgraded above in the tag step.